web/docs/getting-started.md

```diff
@@ -12,8 +12,8 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 ## 1. Requirements
 
 You need to have `node` (and `npm`) installed on your machine and available in `PATH`.
```

Comment:
I think this file was not reverted when the others were.

Reply:
Thanks, I fixed it. GitHub now shows a different unrelated file as different from `main`, but it seems to be a bug on their side.